bigquery/google/cloud/bigquery/job.py

```diff
@@ -2558,6 +2579,18 @@ def to_api_repr(self):
 
         return resource
 
+    def _set_properties(self, api_response):
+        """Update properties from resource in body of ``api_response``
+
+        :type api_response: dict
+        :param api_response: response returned from an API call
```

Comment:
One quick remark while in draft mode - docstrings should use the Google style format, thus let's use it here, too.

I did read that someone is preparing a script to do that automatically, but let's not make this PR dependent on it, or require multiple passes of the script (if e.g. this PR is not merged in time fro the first pass).

Reply:
Thanks for letting me know. I can manually reformat this (and the method in the parent class) before making this PR ready for review.

Reply:
@plamut, @shubha-rajan, yes, yesterday I've got approval from Tim Swast - we're gonna proceed with the script. Not only BigQuery's docstrings should be reformatted, so it is a huge work - thousands of comments to update, and it's better be done automatically